SF iOS/SF iOS/EventDetails/EventDetailsViewController.m

```diff
@@ -171,7 +171,7 @@ - (void)getTravelTimes {
                 }];
             }
         }];
-    }];
+    } shouldQueueIfLocationIsUnavailable:true];
```

Comment:
Seems like we are always queuing. Do we need the additional check? What’s the case where we wouldn’t just always queue?

Reply:
There could potentially be some kind of QoL feature run on an interval, where if the device failed to get the location on any particular invocation the request could just be dropped because another attempt would be made anyway.

Reply:
I guess so, seems a little over engineered but I feel fine merging this in.